### PR TITLE
Feat(Orders) Merged metagenome orderform with others

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
The metagenome orderform is very similar to to the "main" 1508 order form. It was therefore decided to merge them. The 1508 Samplesheet is also update with a new concentration field while the unused(?) quantity field is removed. The concentration not parsed in this PR. This is left for the PR implementing the skip-rc PR

### Changed

- Metagenome orders now go via the 1508 sheet

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/apps/orderform/excel_orderform_parser.py [hunk 1]
```diff
@@ -152,7 +152,14 @@ def get_project_type(self, document_title: str) -> str:
         if Orderform.RML in document_title:
             return str(OrderType.RML) if analysis == self.NO_ANALYSIS else analysis
         if Orderform.MIP_DNA in document_title:
-            return str(OrderType.FASTQ) if analysis == self.NO_ANALYSIS else analysis
+            if analysis == self.NO_ANALYSIS:
+                return (
+                    str(OrderType.METAGENOME)
```
Comment (henrikstranneheim) [inline cg/apps/orderform/excel_orderform_parser.py:157]: Do we need all the str coercion now when python is 3.11 and the enum in StrEnum?

File: cg/constants/orderforms.py [hunk 2]
```diff
@@ -31,9 +31,8 @@ class Orderform(StrEnum):
 
 
 ORDERFORM_VERSIONS = {
```
Comment (henrikstranneheim) [inline cg/constants/orderforms.py:33]: Could we make this a classmethod for Orderform and not a detached constant?

**Review by henrikstranneheim — APPROVED** [review]
👍

File: cg/constants/orderforms.py (hunk 2 above)
Comment (Vince-janv) [reply cg/constants/orderforms.py]: Made it a `static_method`

**Pull request merged**